buildSrc/subprojects/parser/src/test/java/org/apache/calcite/test/DialectGenerateTest.java

```diff
@@ -24,10 +24,13 @@
 import java.util.Arrays;
 import java.util.Map;
 import java.util.LinkedHashMap;
+import java.util.HashSet;
```

Comment:
nit: Out of order alphabetically. Can you also fix the order of the other imports above?